docs/products/postgresql/concepts/upgrade-failover.rst

```diff
@@ -57,3 +57,33 @@ During maintenance updates, cloud migrations, or plan changes, the below procedu
 
 .. Note::
     The old primary server is kept alive for a short period of time (minimum 60 seconds) with a TCP forwarding setup pointing to the new primary server allowing clients to connect before learning the new IP address.
+
+Recreation of replication slots
+-------------------------------
+
+In case of failover or controlled switchover of an Aiven for PostgreSQL service, the replication slots from the old primary server are automatically recreated in the new primary server.
+
+.. important::
+
+    The recreation of replication slots feature is enabled automatically and doesn't require restarting the nodes.
+
+One-node cluster
+""""""""""""""""
+
+Before replacing a node in the one-node cluster, the new node acquires information on replication slots on the original service, re-creates them, and only then the failover is performed.
+
+Multi-node cluster
+""""""""""""""""""
+
+For multi-node setups, replication slots from the primary are synchronized to the standbys periodically. At regular time intervals
+
+* Dependencies for newly-created slots are installed in the corresponding databases (currently, every 30 seconds).
+  When the new slot is created on a database and we want to re-create this slot on a standby, we use a functionality from the ``aiven_extras`` extension, which needs to be installed in the database. Therefore, every 30 seconds there is a job checking that this extension is installed on the databases with logical replication slots.
+* Positions (``confirmed_flush_lsn``) of the slots are synchronized between the primary and the standbys.
+
+When a failover to a standby occurs, the standby node already has active replication slots with an up-to-date (maximum 5-second delay) positions from the primary.
```

Comment:
Please remove word `active` as it can be confusing, replication slot is active in PG when the client is already connected to it.